docs/sources/enterprise/reporting.md

```diff
@@ -131,6 +133,14 @@ concurrent_render_limit = 4
 # Set the scale factor for rendering images. 2 is enough for monitor resolutions
 # 4 would be better for printed material. Setting a higher value affects performance and memory
 image_scale_factor = 2
+# Path to the directory containing font files
+fonts_path =
+# Name of the TrueType font file with Regular style
```

Comment:
```suggestion
# Name of the TrueType font file with regular style
```

What is the difference in meaning between Regular and regular? Similar comment for content in lines 140 and 142.

Reply:
There's no difference in meaning. It's just a font's style. You're right, will change to lowercase.